### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by mnca16**

### CODE REVIEW

File: 01-node-tutorial/answers/week-1/alternative-flavor.js
Comment (IreneS1) [inline 01-node-tutorial/answers/week-1/alternative-flavor.js:1]: I like how you used the student names from names.js here!

File: 01-node-tutorial/answers/week-1/fs-async.js
Comment (IreneS1) [inline 01-node-tutorial/answers/week-1/fs-async.js:1]: Interested in why you nested the writeFile() in the else statements and why three different files? I just wrote the three files to file fileB.txt and appended to the file using the append flag: `{ flag: 'a'}`

File: 01-node-tutorial/answers/week-1/notes.md
Comment (IreneS1) [inline 01-node-tutorial/answers/week-1/notes.md:1]: Love your notes! I'll probably start doing the same starting next lesson hehe

File: 01-node-tutorial/.gitignore
Comment (IreneS1) [inline 01-node-tutorial/.gitignore:1]: Is this to be able to push the files that are in your temporary folder?

**Review by IreneS1 — COMMENTED** [review]
Great job! I need to write more comments on my code like you. Thanks for sharing your wisdom with me (:

File: 01-node-tutorial/answers/week-1/alternative-flavor.js
Comment (mnca16) [reply 01-node-tutorial/answers/week-1/alternative-flavor.js:1]: Thank you!!

File: 01-node-tutorial/answers/week-1/fs-async.js
Comment (mnca16) [reply 01-node-tutorial/answers/week-1/fs-async.js:1]: I think I got confused following the video - some how I though I was getting a response and I wanted to append the response within the next write call. 

I forgot to update it and that's why I added this comment below:

"I attempt to write results from the first and second response but I would need to read what's in the file first"

File: 01-node-tutorial/.gitignore
Comment (mnca16) [reply 01-node-tutorial/.gitignore:1]: Yeah! I think I needed to fix this path so those changes wouldn't be push here

**Review by mnca16 — COMMENTED** [review]
Thank you so much for the review!! We're doing it 😎

File: 01-node-tutorial/answers/week-1/fs-async.js
Comment (IreneS1) [reply 01-node-tutorial/answers/week-1/fs-async.js:1]: Oh okay. They changed it up from what the video was doing, so I totally get why it was confusing!

**Pull request merged**